### PR TITLE
Renaming package names and groupIds

### PR DESCRIPTION
**Pull request by TheMeinerLP**
# **HARD BREAKING CHANGES**
This pull request changes the package name from `de.dytanic.cloudnet*` to `eu.cloudnetservice.v2.*`
Older API,plugins and Modules and more dosen't worke anymore.

### CODE REVIEW

**Comment by TheMeinerLP** [comment]
I have local tested the maven, running fine. But not here ?

**Comment by GiantTreeLP** [comment]
Given that version 2.2 is mostly breaking, forcing everything to be broken, should be fine

**Comment by GiantTreeLP** [comment]
After reassuring with the rest of the team, we would like to use `eu.cloudnetservice.cloudnet.v2` for CloudNet V2

File: Jenkinsfile [hunk 1]
```diff
@@ -34,16 +34,11 @@ pipeline {
         sh 'mvn package'
       }
     }
-    stage('Re-package') {
-      steps {
-        sh 'mvn package javadoc:aggregate-jar -P deployment '
-      }
-    }
```
Comment (GiantTreeLP) [inline Jenkinsfile]: Why has the generation of documentation been removed without replacement?

File: cloudnet-api/cloudnet-api-bridge/pom.xml [hunk 2]
```diff
@@ -22,7 +22,7 @@
 
     <dependencies>
         <dependency>
-            <groupId>de.dytanic.cloudnet</groupId>
+            <groupId>eu.cloudnetservice.cloudnet.v2</groupId>
```
Comment (GiantTreeLP) [inline cloudnet-api/cloudnet-api-bridge/pom.xml:25]: I can't comment on non-changes, but please change the relocation patterns as well

**Pull request merged**